tilebox-datasets/tilebox/datasets/uuid/uuid.py

```diff
@@ -1,35 +1,36 @@
 # allow the uuid module name which shadows the builtin:
```

Comment:
I think it doesn't make sense to have a `uuid.py` file here if the module name is already `uuid` - 

Two options:
- (a) Let's just move this code to `uuid/__init__.py` instead
- (b) Remove the package, and move `uuid.py` outside directly into `tilebox.datasets`

for both the end result is this nicer import:

`from tilebox.datasets.uuid import uuid_message_to_uuid`

I think we go with option (b) for now - since we can anyway always still switch to option a wihtout breaking anything